easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3_no_symlinks_install.patch

```diff
@@ -0,0 +1,25 @@
+don't try to link binaries in /usr/local/bin
```

Comment:
@Serpens Please also mention author or upstream reference to source of this patch

Reply:
Sorry about that